tests/unittest.py

```diff
@@ -317,6 +317,15 @@ def wait_on_thread(self, deferred, timeout=10):
             self.reactor.advance(0.01)
             time.sleep(0.01)
 
+    def wait_for_background_updates(self) -> None:
+        """Block until we've written completed all background updates."""
```

Comment:
I think there's a word missing or something?

```suggestion
        """Block until all background updates have completed."""
```

Reply:
Ahh, thank you. I had in mind something like "block until we've written to the DB that all background updates have completed". But I think the former is an unnecessary detail.

Reply:
Yeah, I don't think the caller really cares about the details, just that they've all finished! (Also, thanks for abstracting this out!)